developer-docs-site/docs/move/move-on-aptos/cryptography.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Cryptography"
+slug: "cryptography"
+---
+
+# Cryptography in Move
+
+The Move language originally supported only four basic cryptographic operations:
+
+1. Hashing with SHA2-256 via [`std::hash::sha2_256`](https://github.com/aptos-labs/aptos-core/blob/137acee4c6dddb1c86398dce25b041d78a3028d3/aptos-move/framework/move-stdlib/sources/hash.move#L6)
+2. Hashing with SHA3-256 via [`std::hash::sha3_256`](https://github.com/aptos-labs/aptos-core/blob/137acee4c6dddb1c86398dce25b041d78a3028d3/aptos-move/framework/move-stdlib/sources/hash.move#L7)
+3. Verifying an Ed25519 signature via (the now obsolete) [`signature::ed25519_verify`](https://github.com/aptos-labs/aptos-core/blob/3b53225b5a1effcce5dee9597a129216510dc424/aptos-move/framework/aptos-framework/sources/Signature.move#L19)
+4. Validating an Ed25519 public key via (the now obsolete) [`signature::ed25519_validate_pubkey`](https://github.com/aptos-labs/aptos-core/blob/3b53225b5a1effcce5dee9597a129216510dc424/aptos-move/framework/aptos-framework/sources/Signature.move#L9)
+
+Since then, we have heavily extended support for cryptography in Move. 
+Behind these new cryptographic features were two **key guiding principles**:
```

Comment:
This feels superfluous. Though I am proud of the work made to get us to where we are today.

Reply:
Maybe just say...

The Aptos adapter for Move has introduced many useful cryptographic primitives. The key guiding principles behind this are

(I don't think we need bold / emphasis)